controllers/machinehealthcheck_controller.go

```diff
@@ -299,10 +299,12 @@ func (r *MachineHealthCheckReconciler) PatchHealthyTargets(ctx context.Context,
 			// Get remediation request object
 			obj, err := r.getExternalRemediationRequest(ctx, m, t.Machine.Name)
 			if err != nil {
-				if apierrors.IsNotFound(errors.Cause(err)) {
-					continue
+				if !apierrors.IsNotFound(errors.Cause(err)) {
+					wrappedErr := errors.Wrapf(err, "failed to fetch remediation request for machine %q in namespace %q within cluster %q", t.Machine.Name, t.Machine.Namespace, t.Machine.ClusterName)
```

Comment:
I have a question about error wrapping, isn't the recommended way to wrap errors after go 1.13(https://blog.golang.org/go1.13-errors) is using `fmt.Errorf()` ? If so, can we prefer it over `errors.Wrapf()` ?

Reply:
Usually we don't use stdlib errors, but pkg/error instead

Reply:
@vincepri is that for historical reasons?

Reply:
Partially, yes. Most of the k/k repositories use pkg/errors